src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -321,6 +317,56 @@ private function updateGetOperation(\ArrayObject $pathOperation, array $mimeType
     }
 
     /**
+     * Returns an array of swagger path parameters according to the provided class' identifiers
+     *
+     * @param string $resourceClass
+     * @return array
+     */
+    private function getPathParameters(string $resourceClass)
+    {
+        $parameters = [];
+        $properties = $this->getClassIdentifiers($resourceClass);
+        foreach ($properties as $propertyName => $property) {
+            $parameters[] = [
+                'name' => $propertyName,
+                'in' => 'path',
+                'required' => true,
+                'type' => $property->getType()->getBuiltinType(),
+            ];
+        }
+
+        if (empty($parameters)) {
+            $parameters = [[
+                'name' => "id",
+                'in' => 'path',
+                'required' => true,
+                'type' => "string",
+            ]];
+        }
+
+        return $parameters;
+    }
+
+    /**
+     * Returns all the identifier properties for the given class
+     *
+     * @param string $resourceClass
+     * @return PropertyMetadata[]
+     */
+    private function getClassIdentifiers(string $resourceClass)
```

Comment:
Can you follow the same strategy as https://github.com/api-platform/core/pull/1358/files#diff-6ad90c31c0386b8ef6b541a28614c002?